dbt_platform_helper/commands/pipeline.py

```diff
@@ -50,7 +50,7 @@ def generate():
 
     codestar_connection_arn = get_codestar_connection_arn(app_name)
     if codestar_connection_arn is None:
-        abort_with_error("There is no CodeStar Connection to use")
+        abort_with_error(f'There is no CodeStar Connection named "{app_name}" to use')
```

Comment:
Made the error message more helpful because I had to think about it too much.

Reply:
Was there a test that needed to change because the error message changed?

Reply:
I changed the test first: https://github.com/uktrade/platform-tools/pull/431/files#diff-7a00acf1fa588a66be7746be27e917251534f6a6dd99223d60d7a1736b2ff00eR198